charts/rancher-webhook/Chart.yaml

```diff
@@ -9,3 +9,4 @@ annotations:
   catalog.cattle.io/namespace: cattle-system
   catalog.cattle.io/release-name: rancher-webhook
   catalog.cattle.io/os: linux
+  catalog.cattle.io/rancher-version: '>= 2.5.10-0 <2.5.99-0'
```

Comment:
Based on our conversation on the other PR, I believe we want to remove the `-0` on `2.5.99`. That way, if we somehow get to `2.5.99`, then this will still get installed.